src/redisearch_api.c

```diff
@@ -179,6 +179,18 @@ void RediSearch_DocumentAddFieldNumber(Document* d, const char* fieldname, doubl
   Document_AddFieldC(d, fieldname, buf, len, as);
 }
 
+void RediSearch_DocumentAddFieldGeo(Document* d, const char* fieldname, 
+                                    double lat, double lon, unsigned as) {
+  if (lat > GEO_LAT_MAX || lat < GEO_LAT_MIN || lon > GEO_LONG_MAX || lon < GEO_LONG_MIN) {
+    // out of range
+    return;
```

Comment:
We can at least say "sorry about that".

Reply:
Changed signature to int and expanded tests